jbpm-console-ng-generic/jbpm-console-ng-data-filters-client/src/test/java/org/jbpm/console/ng/df/client/filter/FilterSettingsJSONMarshallerTest.java

```diff
@@ -0,0 +1,149 @@
+import org.dashbuilder.dataset.filter.FilterFactory;
+import org.dashbuilder.dataset.group.DataSetGroup;
+import org.dashbuilder.dataset.json.DataSetJSONMarshaller;
+import org.dashbuilder.dataset.json.DataSetLookupJSONMarshaller;
+import org.dashbuilder.dataset.sort.DataSetSort;
+import org.dashbuilder.displayer.json.DisplayerSettingsJSONMarshaller;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.dashbuilder.dataset.filter.FilterFactory.*;
+import static org.dashbuilder.dataset.sort.SortOrder.*;
+import static org.junit.Assert.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FilterSettingsJSONMarshallerTest {
+
+    public static final String COLUMN_ACTIVATIONTIME = "activationTime";
+    public static final String COLUMN_ACTUALOWNER = "actualOwner";
+    public static final String COLUMN_CREATEDBY = "createdBy";
+    public static final String COLUMN_CREATEDON = "createdOn";
+    public static final String COLUMN_DEPLOYMENTID = "deploymentId";
+    public static final String COLUMN_DESCRIPTION = "description";
+    public static final String COLUMN_DUEDATE = "dueDate";
+    public static final String COLUMN_NAME = "name";
+    public static final String COLUMN_PARENTID = "parentId";
+    public static final String COLUMN_PRIORITY = "priority";
+    public static final String COLUMN_PROCESSID = "processId";
+    public static final String COLUMN_PROCESSINSTANCEID = "processInstanceId";
+    public static final String COLUMN_PROCESSSESSIONID = "processSessionId";
+    public static final String COLUMN_STATUS = "status";
+    public static final String COLUMN_TASKID = "taskId";
+    public static final String COLUMN_WORKITEMID = "workItemId";
+    public static final String COLUMN_ORGANIZATIONAL_ENTITY = "oeid";
+
+    @Mock
+    protected DataSetLookup dataSetLookup;
+
+
+    protected DataSetSort datasetSort;
+    protected DataSetFilter datasetFilter;
+    protected DataSetGroup datasetGroup;
+
+    FilterSettingsJSONMarshaller filterSettingsJSONMarshaller;
+
+    @Before
+    public void setUp() throws Exception {
+        DataSetJSONMarshaller datasetJsonMarshaller = new DataSetJSONMarshaller();
+        DataSetLookupJSONMarshaller lookupJsonMarshaller = new DataSetLookupJSONMarshaller();
+        DisplayerSettingsJSONMarshaller displayerJsonMarshaller = new DisplayerSettingsJSONMarshaller(datasetJsonMarshaller, lookupJsonMarshaller);
+        filterSettingsJSONMarshaller = new FilterSettingsJSONMarshaller(displayerJsonMarshaller);
+    }
+
+    @Test
+    public void testJSONConversion() throws Exception {
+        FilterSettings filterSettings = createTableSettings();
+        String json = filterSettingsJSONMarshaller.toJsonString(filterSettings);
+        FilterSettings fs = filterSettingsJSONMarshaller.fromJsonString(json);
+        assertTrue("Check the number of operations",
```

Comment:
+1 for the comment (have clear asserts)... but due the emergency (master broken), I'm considering OK for merge.